quiz.go

```diff
@@ -38,10 +39,10 @@ func main() {
 		fmt.Scanln(&response)
```

Comment:
How cool is this? I couldn't really understand what `scanLn` returns, an `int` and an `err`? What is the int for?